### DISCUSSION TITLE
ERRORE: Default_config

### DISCUSSION THREAD

**Discussion by Ilmessaggeromascherato** [indomus/forum#2441] [HUB: Home Assistant]
Salve, 
ho fatto l'aggiornamento di HA qualche settimana fa dalla 2023.3 alla 2023.4 e ora mi compare questo errore: 
Setup failed for stream: Requirements for stream not found: ['ha-av==10.0.0'].

ho provato di tutto ma non riesco a risolverlo. 
Qualcuno mi può aiutare ? 
il mio HA è installato su un RaspBerry PI4. 
Grazie mille

**Comment by Scialla** [comment]
Se usi HA installato in Venv, dovrai sistemarti a mano tutte queste cose, con docker questo non accade. Il consiglio è quindi di passare a docker.
Se vuoi restare con Venv devi essere in grado di risolvere questi problemi autonomamente, oppure cercare soluzioni in rete.
Google è tuo amico
https://community.home-assistant.io/t/error-ha-av-with-2023-04/558497
